engine/hooks/hooks_handlers.go

```diff
@@ -399,13 +400,17 @@ func (s *Service) postTaskBulkHandler() service.Handler {
 		}
 
 		for _, hook := range hooks {
-			if hook.UUID != "" {
-				if err := s.deleteTask(ctx, &hook); err != nil {
-					log.Warning("Hooks> postTaskBulkHandler cannot delete task %s : %v", hook.UUID, err)
+			if hook.UUID == "" {
```

Comment:
UUID must not be empty, insertNode set UUID

Reply:
done